src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.cs

```diff
@@ -343,7 +342,7 @@ private async Task SelectAndStartTransport(TransferFormat transferFormat, Cancel
                 }
 
                 // This should only need to happen once
-                var connectUrl = CreateConnectUrl(uri, _connectionToken);
+                var connectUrl = CreateConnectUrl(uri, negotiationResponse.ConnectionToken);
```

Comment:
This URL isn't exposed anywhere, right?

Reply:
It does get logged at the debug level

Reply:
That's OK

Reply:
Well, I think it is. We could also redact it from the logs.

@blowdart @GrabYourPitchforks do you have any thoughts here on if the connection token value should be suppressed from `Debug` level logs on the **Client**.